### PR TITLE
chore: update licence year

### PR DESCRIPTION
**Pull request by joaoandre-avaiga**

### CODE REVIEW

**Review by jrobinAV — DISMISSED** [review]

File: tests/conftest.py [hunk 1]
```diff
@@ -165,11 +165,12 @@ def __default_task():
         properties={"default_data": "In memory Data Source"},
     )
     return Task(
-        "foo",
-        print,
-        [input_ds],
-        [output_ds],
-        None,
+        config_id="foo",
```
Comment (jrobinAV) [inline tests/conftest.py:168]: Indeed, it is my bad. 
I moved in taipy-core the parameter `properties` from the last position to the second one. 

Thx joao!

File: tests/json/expected/cycle.json [hunk 2]
```diff
@@ -7,5 +7,6 @@
     },
     "creation_date": "2022-03-31T21:50:52.349698",
     "start_date": "2022-03-31T00:00:00",
-    "end_date": "2022-03-31T23:59:59.999999"
+    "end_date": "2022-03-31T23:59:59.999999",
+    "version": "1.0"
```
Comment (jrobinAV) [inline tests/json/expected/cycle.json]:
```suggestion
```

I believe cycle does not have a version attached. Right?

Comment (joaoandre-avaiga) [reply tests/json/expected/cycle.json]: 👍

File: src/taipy/rest/version.py [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+# Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+# an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+import json
+import os
+
+
+def _get_version():
+    with open(f"src{os.sep}taipy{os.sep}rest{os.sep}version.json") as version_file:
```
Comment (jrobinAV) [inline src/taipy/rest/version.py]:
```suggestion
    with open(f"{os.path.dirname(os.path.abspath(__file__))}{os.sep}version.json") as version_file:
```

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**